apigentools/cli.py

```diff
@@ -1,7 +1,7 @@
 # Unless explicitly stated otherwise all files in this repository are licensed
 # under the 3-clause BSD style license (see LICENSE).
 # This product includes software developed at Datadog (https://www.datadoghq.com/).
-# Copyright 2019 Datadog, Inc.
+# Copyright 2020 Datadog, Inc.
 import argparse
```

Comment:
I am adding line breaks after the header for all files.

Reply:
I personally prefer no line break, but don't have a strong opinion if we want them